tests/irepela/homework_7/test_data.py

```diff
@@ -0,0 +1,2 @@
+def my_func(a):
+    return a * 2
```

Comment:
actually you can put it to test directly, 

you can even use lambda for this purpose, it is 1 line of code:


my_func = lambda a: a * 2